### ISSUE TITLE
iPhone/iPad issues

### ISSUE THREAD

**Issue by davidatfuzzylime** [cheeaun/mooeditable#41]
The editor doesn't work in mobile browsers, including those on the iPhone/iPad, at the moment. You can tap the button to edit in HTML view but the main iframe version doesn't work at all, unfortunately.

**Comment by ryanmitchell** [comment]
That is due to the limitations of the iOS platform - it doesn't support content editable.
Probably should just fallback to textarea?

**Comment by davidatfuzzylime** [comment]
Yeah I feared that was the case. I guess a fallback would be the best option in the meantime!

**Comment by viktors** [comment]
On iOS5 it might work fine: http://www.youtube.com/watch?v=I3NQlwBOXpQ

**Comment by marcj** [comment]
Yepp, I checked my fork of mooeditable ('cause the demo from cheeaun is corrupt) few minutes ago with iOS 5 Beta - looks good, but needs some work in the text selection area and image selection (especially resizing).
